### ISSUE TITLE
Essential, inseparable traits are not persisted

### ISSUE THREAD

**Issue by tomasara413** [OpenPonk/class-editor#38]
Saved assigned traits do not appear after loading a project file. Either the traits are not being saved or are not loading from the file.
The problem is apparent in all relations which do allow these traits to be checked - «ComponentOf», «MemberOf», «SubQuantityOf» and «SubCollectionOf» relations.

**Comment by JanBliznicenko** [comment]
Hello, could you, please, take your "image" directory in OpenPonk, put it into an archive (zip, rar, 7z...) and upload here to this issue (or anywhere else)? I will try to investigate, as I was not able to reproduce the issue on my PC.
Also, try current nightly build and check whether it has the same issue...
(Onto)UML build: https://github.com/OpenPonk/class-editor/releases/tag/nightly
Build with all OP plugins: https://github.com/OpenPonk/plugins/releases/tag/nightly

Thank you

**Comment by tomasara413** [comment]
It indeed does not work on either build, previously I tried to test it on the all plugins, but with the former link the traits were not saved either. As per request the files are listed below (I was unable to attach zip directly). I also attached a video of this issue and the project file on which the tests were performed on (other project still suffer with this problem). The issue is present on windows and linux (I do not have mac to test), although I did not test the linux distribution with the class editor release, only all plugins which realeased week ago.

Class editor image: https://drive.google.com/file/d/18K0fX55zM6LOrTjyynsVx1IgIzUVpF-U/view?usp=sharing
All plugins: https://drive.google.com/file/d/1iufXG6YHe0QUY-Z42064DTUxttLMb2MX/view?usp=sharing
Project file: https://drive.google.com/file/d/1s0Zh49pk-Snl5j5gA7VRhcxLEgK4POIu/view?usp=sharing

https://user-images.githubusercontent.com/1576436/140950546-fa00932c-f600-425c-b429-d3e48b30b3f6.mp4
